### PR TITLE
update logger for autoenticator.go, passed on docker-compose build

### PR DESCRIPTION
**Pull request by wy65701436**
update the logger for this go file, and delete bee import as the build failure.

### CODE REVIEW

**Comment by reasonerjt** [comment]
LGTM

**Comment by ywk253100** [comment]
LGTM

File: auth/authenticator.go [hunk 1]
```diff
@@ -17,12 +17,11 @@ package auth
 
 import (
 	"fmt"
-	"log"
 	"os"
 
-	"github.com/vmware/harbor/models"
+	log "github.com/vmware/harbor/utils/log"
```
Comment (reasonerjt) [inline auth/authenticator.go]: "github.com/vmware/harbor/utils/log" should be Good, no need to rename

File: auth/ldap/ldap.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func (l *Auth) Authenticate(m models.AuthModel) (*models.User, error) {
 		return nil, err
 	}
 	if len(result.Entries()) != 1 {
-		log.Printf("Found more than one entry.")
+		log.Infof("Found more than one entry.")
```
Comment (reasonerjt) [inline auth/ldap/ldap.go]: I think this is a warning.

**Pull request merged**